scripts/lint_site.sh

```diff
@@ -151,17 +151,20 @@ done
 if [[ ${#SKIP_LANGS[@]} -ne 0 ]]; then
     printf -v find_exclude " -name %s -prune -o" "${SKIP_LANGS[@]}"; read -r -a find_exclude <<< "$find_exclude"
 fi
-while IFS= read -r -d '' f; do
-    if grep -H -n -i -e blockquote "${f}"; then
-        echo "Ensure content only uses {{< tip >}}, {{< warning >}}, {{< idea >}}, and {{< quote >}} instead of block quotes"
-        FAILED=1
-    fi
 
-    #if grep -H -n -e "\"https://github.*#L[0-9]*\"" "${f}"; then
-    #    echo "Ensure content doesn't use links to specific lines in GitHub files as those are too brittle"
-    #    FAILED=1
-    #fi
-done < <(find ./public "${find_exclude[@]}" -type f -name '*.html' -print0)
+if [ -d ./public ]; then
```

Comment:
Actually, this could move up a few lines too, because line 151 isn't needed either, if there's no ./public.

Reply:
Yes.  I'll send out a revised version.  Thanks!